addons/website_sale_loyalty/models/sale_order.py

```diff
@@ -131,7 +131,6 @@ def _compute_website_order_line(self):
                     'discount': 0.0,
                     'name': lines[0].name_short if lines.reward_id.reward_type != 'product' else lines[0].name,
                     'product_uom_qty': 1,
-                    'product_uom': lines[0].product_uom.id,
```

Comment:
This is not the same, it's the uom selected on the line, not the uom of the product of the line.
It could be different